brewchat/src/main/java/com/example/brewchat/activities/BaseActivity.java

```diff
@@ -18,4 +26,26 @@ protected void onStop() {
         EventBus.getDefault().unregister(this);
     }
 
+    @SuppressWarnings("unused")
+    public void onEventMainThread(UserLogoutEvent event) {
+        AlertDialog.Builder alertDialog = new AlertDialog.Builder(this);
```

Comment:
I would put this into an specific method. Something like `displayLogoutConfimationDialog`